### ISSUE TITLE
Not catching HTML instead of JSON

### ISSUE THREAD

**Issue by Oreolek** [RijkvanZanten/directus-sdk-node#1]
I think it's not catching JSON parsing errors properly somewhere. (I use the self-hosted CMS version)

```
undefined:1
<!doctype html>
^

SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at Request.r [as _callback] (/www/oreolek6/frontend-metalsmith/node_modules/directus-sdk-node/index.js:423:292)
    at Request.self.callback (/www/oreolek6/frontend-metalsmith/node_modules/request/request.js:186:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/www/oreolek6/frontend-metalsmith/node_modules/request/request.js:1081:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/www/oreolek6/frontend-metalsmith/node_modules/request/request.js:1001:12)
    at IncomingMessage.g (events.js:292:16)
FAIL: 1
```

**Comment by RijkvanZanten** [comment]
That (of course) shouldn't be happening. When did this exact error occur?

**Comment by Oreolek** [comment]
I can't place the exact line but here's my script:

```js
const directus = require('directus-sdk-node');

<..getting the config..>

const client = new directus(config.directus_token, {
  baseUrl: config.directus_url,
  apiVersion: 1 // Optional - default 1
});

client.getEntries('posts', (err, res) => {
  if (err) {
    console.log (err);
    throw (err);
  }
});


```

**Comment by Oreolek** [comment]
upd: the error was in my config, just needed to set url not as `<url>` but as `<url>/api`. Still it's a weird unexplainable crash.

**Comment by RijkvanZanten** [comment]
Yeah, I will add a check to check if you provided the /api or not and add it if it isn't there. Thanks for the heads up!

**Comment by RijkvanZanten** [comment]
Fixed the `/api`issue in version `v1.1.0`!
